src/manifests/ingress/nginx/nginx.yaml

```diff
@@ -90,21 +90,6 @@ rules:
   - get
   - list
   - watch
-- apiGroups:
```

Comment:
Has latest nginx moved to leases approach? is this the main reason we don't need this rule anymore?

Reply:
I don't know, I've just copy/paste their manifest. What rule?